### DISCUSSION TITLE
New code view does not use anchors in sidebar markup

### DISCUSSION THREAD

**Discussion by nwalters512** [community/community#41537] [Repositories]
### Select Topic Area

Product Feedback

### Body

The new React-powered code view in repositories doesn't use `<a>` elements for files in the sidebar, which means I can't right-click and open a files in a new tab in order to view multiple files at the same time. Since clicking a file does indeed navigate you to a new URL, it's semantically correct to use `<a>` in this case.

**Comment by ForsakenHarmony** [comment]
This is insanely frustrating, I middle-click almost every link and here I have to duplicate the tab and then select there if I want to open a second file
